@AH-64D Apache Longbow/Addons/fza_ah64_mpd/CfgVehicles.hpp

```diff
@@ -139,6 +139,8 @@ class CfgVehicles {
                 usesIcons = 1;
                 class InitState {
                     subPageVarPage[] = {5,0};
+                    arrowL = 0;
```

Comment:
On discussion, the arrow swivelling logic does not look functional right now. Please can you remove until it is usable? Or at least not have in this PR seeing as it is not integral.

I would recommend leaving it in a PR though so we don't lose it

Reply:
corrected